modules/gitops-uninstall-keycloak.adoc

```diff
@@ -0,0 +1,23 @@
+[id="uninstalling-keycloak_{context}"]
+= Uninstalling Keycloak
+
+You can delete the Keycloak resources and its relevant configuration by removing the SSO field from Argo CD Custom Resource Spec. After removing, the SSO field will look like as shown below.
```

Comment:
Let's rephrase the last sentence to avoid future tense and the spacial "below." For accessibility, we try to avoid these visual works like "below, above" or colors, like, "the green arrow button." 

`After removing, the SSO field looks like the following:`